docs/en/userguide/index.md

```diff
@@ -1,26 +1,24 @@
 title: CKAN Registry
 summary: Display data from an external CKAN resource
 
-# CKAN Registry
-
-## Table of contents
+## Features
 
 * [Loading resource data](#loading-resource-data)
-  * [Changing resource data](#changing-resource-data)
-  * [Configuring data columns](#configuring-data-columns)
+* [Changing resource data](#changing-resource-data)
+* [Configuring data columns](#configuring-data-columns)
 * [Configuring how fields are displayed](#fields)
 * [Configuring how users can filter data](#filters)
-  * [Configuring a text filter](#configuring-a-text-filter)
-  * [Configuring a dropdown filter](#configuring-a-dropdown-filter)
-    * [Select from all options](#dropdown-options)
-    * [Splitting delimited cell values](#dropdown-delimiter)
-      * [Manual selection](#dropdown-manual-selection)
+* [Configuring a text filter](#configuring-a-text-filter)
+* [Configuring a dropdown filter](#configuring-a-dropdown-filter)
+* [Select from all options](#dropdown-options)
+* [Splitting delimited cell values](#dropdown-delimiter)
+* [Manual selection](#dropdown-manual-selection)
```

Comment:
Did we remove the sections of this TOC because docsviewer has issues with it? I think that 
"Configuring a text filter" is definitely a sub-title off "Configuring how users can filter data".

Reply:
The styles in userhelp.silverstripe.org for bullet points are a little off padding wise. I agree but even think these sections are really in-depth and we could actually remove some of them as someone could use the TOC that's provided anyway above. Thoughts?

Reply:
FYI I'll make a note to look at the padding of bullet points 😄